app/db/models/document/physical_document.py

```diff
@@ -0,0 +1,45 @@
+import sqlalchemy as sa
+from app.db.session import Base
+
+
+class PhysicalDocument(Base):
+    """A physical document.
+
+    Representation of a document that exists in the real world.
+    Owned and updated only by the pipeline.
+    """
+
+    __tablename__ = "physical_document"
+
+    id = sa.Column(sa.Text, primary_key=True)
```

Comment:
I'm not sure why we're setting the primary key to be text at this point; we have nothing sensible to set it to so why not use int?

Reply:
Is that not pushing law/policy domain information into the application agnostic document schema layer?

Reply:
also... we have a single "application schema" at the moment, and while it's true that the family-document <-> physical-document is 1:1 for that app, can we guarantee that constraint holds for all future apps?

Reply:
1) It's not really pushing law/policy into this layer - its allowing a upper layer to reference the document schema in a way that it understands - thinking of document id as an abstract identifier that is actually defined in the upper layer... if that makes sense

2) This also re-inforces the above idea - in that even in other apps where the reln is not 1:1 ... the definition of the id needs to really be specified at the upper-scheama level

Reply:
FTR - conversation concluded to use an ID